static/css/nl_interface.scss

```diff
@@ -206,6 +225,15 @@ $answer-min-height: 100vh;
   }
 }
 
+.nl-query h2 {
+  color: $question-text-color;
+}
+
+.form-control:focus {
+  // Override the red border which looks like an error.
+  border-color: var(--dark) !important;
```

Comment:
maybe also remove the box-shadow/outline because it still looks a little red? https://github.com/datacommonsorg/website/blob/master/static/css/import_wizard2.scss#L38-L43

Reply:
how about adding it as an override to base.scss?

Reply:
(kept the box shadow but altered the color - it's a nice visual feedback for where the focus is)

![image](https://user-images.githubusercontent.com/6052978/221017691-01d1225c-fdd9-40cc-99f3-19c155619def.png)
![image](https://user-images.githubusercontent.com/6052978/221017740-eee1c127-9d78-4770-af7f-9bb9064b92e8.png)

Reply:
yes, that looks good!